.travis.yml

```diff
@@ -301,9 +301,9 @@ install:
     fi
   - if [ $USE_PYTHON == "ON" ]; then
       travis_wait spack install
-        py-pybind11@2.2.1 ^python@$TRAVIS_PYTHON_VERSION
+        python@$TRAVIS_PYTHON_VERSION
         $COMPILERSPEC &&
-      spack load --dependencies py-pybind11@2.2.1 ^python@$TRAVIS_PYTHON_VERSION $COMPILERSPEC;
+      spack load python@$TRAVIS_PYTHON_VERSION $COMPILERSPEC;
```

Comment:
> We don't need to "install" pybind11 beforehand now.

Reply:
maybe we should abuse one to install the *latest* pybind11 version (since we ship the oldest supported version)?

Reply:
Have to find a creative idea how to add extra cmake flags without being super-verbose.

Reply:
@C0nsultant fixed :)